### PR TITLE
HAAR-549: Allow non alpha chars

### PR DESCRIPTION
**Pull request by ravmoj**
Allow Non-Alpha chars('’-) in first and last name

### CODE REVIEW

File: backend/controllers/createDpsUser.js [hunk 1]
```diff
@@ -88,6 +87,10 @@ const createDpsUserFactory = (getCaseloads, createDpsUser, createUserUrl, manage
   return { index, post }
 }
 
+function removeForwardApostrophe(str) {
```
Comment (karenmillermoj) [inline backend/controllers/createDpsUser.js:90]: This would probably sit neater in utils.js

Comment (ravmoj) [reply backend/controllers/createDpsUser.js]: Done

File: backend/controllers/dpsUserValidation.test.js [hunk 2]
```diff
@@ -68,6 +81,34 @@ describe('DPS create validation', () => {
       ),
     ).toEqual([])
   })
+
+  it('should pass fist and last name validation with ', () => {
```
Comment (karenmillermoj) [inline backend/controllers/dpsUserValidation.test.js:85]: fiRst typo

Comment (ravmoj) [reply backend/controllers/dpsUserValidation.test.js]: updated to first

File: backend/controllers/dpsUserValidation.test.js [hunk 3]
```diff
@@ -47,11 +47,24 @@ describe('DPS create validation', () => {
...
+      ]),
+    )
+  })
+
+  it('should validate specific characters allowed for Last name field with alphabetical strings', () => {
+    expect(validateDpsUserCreate('un', 'b@d.com', 'ca', 'de3', 'default', true, 'Select a default caseload')).toEqual(
+      expect.arrayContaining([
+        {
+          href: '#lastName',
+          text: 'Last name must be consist of letters, an forward apostrophe, standard apostrophe & hyphen only',
```
Comment (karenmillermoj) [inline backend/controllers/dpsUserValidation.test.js:66]: same as above

Comment (ravmoj) [reply backend/controllers/dpsUserValidation.test.js]: Updated message

File: backend/controllers/dpsUserValidation.test.js [hunk 4]
```diff
@@ -47,11 +47,24 @@ describe('DPS create validation', () => {
...
   })
-  it('should validate specific characters allowed for alphabetical strings', () => {
+  it('should validate specific characters allowed for First name field with alphabetical strings', () => {
     expect(validateDpsUserCreate('un', 'b@d.com', 'ca4', 'de3', 'default', true, 'Select a default caseload')).toEqual(
       expect.arrayContaining([
-        { href: '#firstName', text: 'First name must be consist of letters only' },
-        { href: '#lastName', text: 'Last name must be consist of letters or an apostrophe only' },
+        {
+          href: '#firstName',
+          text: 'First name must be consist of letters, an forward apostrophe, standard apostrophe & hyphen only',
```
[the remark this answers is not in the document]
Comment (karenmillermoj) [reply backend/controllers/dpsUserValidation.test.js]: This still doesn't read well, First name must be consist of letters, an forward apostrophe, standard apostrophe & hyphen only

**Pull request merged**